pkg/views/gitprovider/select.go

```diff
@@ -136,7 +136,24 @@ func providerRequiresUsername(gitProviderId string) bool {
 }
 
 func providerRequiresApiUrl(gitProviderId string) bool {
-	return gitProviderId == "gitness" || gitProviderId == "github-enterprise-server" || gitProviderId == "gitlab-self-managed" || gitProviderId == "gitea" || gitProviderId == "bitbucket-server" || gitProviderId == "azure-devops" || gitProviderId == "aws-codecommit"
+	providersRequiringApiUrl := []string{
+		"gitness",
+		"github-enterprise-server",
+		"gitlab-self-managed",
+		"gitea",
+		"bitbucket-server",
+		"azure-devops",
+		"aws-codecommit",
+		"gogs",
+	}
+
+	for _, provider := range providersRequiringApiUrl {
```

Comment:
I believe we could simplify with `slices.Contains`

Reply:
done